examples/set_colormaps.py

```diff
@@ -16,7 +16,7 @@
 blue = vispy.color.Colormap([[0., 0., 0.], [0., 0., 1.]])
 
 with napari.util.app_context():
-    v = napari.Viewer()
+    v = napari.ViewerApp()
```

Comment:
In line 8 of this file, should the import now just be `import napari` not `import napari.util`? It doesn't seem to make a difference which confuses me.

Reply:
I have changed it to `import napari` as it looks cleaner to me, hope that's ok